Moves security to deck 1, makes deck 3 more habitable

CakeQ opened the pull request:
🆑 Cakey
maptweak: The brig has been moved to deck one. Deck three has been adjusted to be further enforce it being the habitation deck.
maptweak: Added the Commissary, an on-ship store run by supply and service crew.
maptweak: Moved the pathfinder's office to deck four.
maptweak: Moved the bluespace drive to deck two.
maptweak: Moved the observation bubble, diplomatic quarters and vacant office to deck three.
/🆑

babydoll commented on maps/torch/job/access.dm:
```diff
@@ -135,6 +135,11 @@
 	region = ACCESS_REGION_RESEARCH
 
+/var/const/access_commissary = 96
+/datum/access/pathfinder
```
This doesn't seem correct

CakeQ replied:
haha shit

babydoll reviewed (commented):
Alright
### torch-4.dmm
- Why are you replacing the deck 1 teleporter with yet another checkpoint?
- Pipe in space on deck 1https://i.imgur.com/7GQhZrP.png
- There's a blue landmark marker missing here, also deck 1: https://i.imgur.com/YBUSfYv.png
### torch-3.dmm
- Where's this ladder go to https://i.imgur.com/6YzgbAd.png
- What's this hole for https://i.imgur.com/Jum8gSv.png
### torch-2.dmm
- Any booze or smokes or whatever that spawns in the store should probably spawn in the back room instead of the place that will be always looted.

PurpleMartinJCK commented:
Also for that ladder, having floor decals under ladders doesn't make sense as the floor is removed.

Sbotkin commented:
Brig belongs to habitation deck, not operation imo.

>maptweak: Added the Commissary, an on-ship store run by supply and service crew.

But why?

CrimsonShrike commented:
if the armoury needs a sec checkpoint of sorts, turning armoury desk into a sec accessible zone would seem to solve the issue without adding a second checkpoint to deck 1

Miraviel commented:
> maptweak: The brig has been moved to deck one. Deck three has been adjusted to be further enforce it being the habitation deck.

I agree with @Sbotkin, the brig belongs to the habitation deck. It's a much better place strategically as well, security can get to areas that are likely to be problematic sooner. (Deck 3 mess hall + EVA, entire Deck 4) 

B-Deck and Deck 1 are already safe as they are (bridge, emergency armory, NT sec, etc.), having the "sec HQ" on deck 3 further balances the overall security of the ship.

> maptweak: Added the Commissary, an on-ship store run by supply and service crew.

This makes the merchant role less appealing. Merchant is a more or less unique role and if we have something onboard like a shop already, people will less likely bother going there. If you think supply needs more things to do, I suggest spicing up their gameplay! Not to subtract from someone else's.

> maptweak: Moved the bluespace drive to deck two.

This will never let the SM fall on the drive ever again, removing the chance of a round-ending catastrophe. It also makes more sense to me to have the drive right below the engine.

babydoll commented:
Let us get the technical things out of the way before this thread gets overrun with lore and conceptual stuff, please.

I flew around the map as a ghost and was almost constantly hearing the pipe leak sound. You need to trace all the paths you made and ensure the pipes are connecting properly.

BlueNexus commented:
I strongly disagree with the changes made here, but I'll refrain from stating them until technical stuff is sorted out.

PurpleMartinJCK commented:
There's a map debugging tool that helps find disconnected pipes

CakeQ commented:
For those of you that are confused: There is only one checkpoint on deck one. I swapped the teleporter and checkpoint locations around so that there is a checkpoint next to the earm, and the teleporter is where the old D1 checkpoint is.
@sabiram ladder issues in engineering aren't there anymore, idk why the mapmerger hasnt updated
Also yes pipe leakages are unavoidable upon first PR. I heavily rely on Travis to find that stuff for me

BlueNexus commented:
You seem to have forgotten the point of those checkpoints. They're meant to be at the stairs, with the ability to prevent access to the fore part of the deck. If you're going to change it for one deck, you should change it for all of them, to avoid inconsistency.

CakeQ commented:
@BlueNexus Uh, no. Checkpoints as a feature are in place to oversee key points within the ship. Deck one's checkpoint was moved to the aft to oversee the emergency armory, and there's no need for the fore checkpoint anymore because the brig is literally right there. All other decks have checkpoints at the stairwell.

babydoll commented:
- Bluespace drive room has an air alarm but is part of an area behind a wall, will give inaccurate alarms

- There's a fuel pipe (I think) leak in front of the armoury:
```
Results of the analysis of the deck:
Pressure: 265.7 kPa
Oxygen: 8%
Nitrogen: 30%
Carbon Dioxide: 52%
Hydrogen: 10%
Temperature: 20°C / 293K
```
- There's a leaking pipe in the north pod of the armoury
- The armoury is absolutely gargantuan
- No need for there to be a checkpoint in front of the armoury AND the little desk gimmick

- Still oodles of pipe leaks in security
- The big brig looks very visually busy
- The outer door leading into the brig is a different type than the one leading out 
- Several pipe leaks outside the conference room

- More pipe leaks in habitation
- Door leading south into maintenance out of the diplomatic suite is incorrectly named

- Cigarette vending machine in the bar is in front of a window
- Blue dining room has no lights
- Not sure about the entire concept of the shop thing TBH
- More pipe leaks in halls outside 3rd deck teleporter

- Pathfinder does not need two lockers

CakeQ commented:
@sabiram Fixed leaky pipes, removed second pathfinder closet, added missing lights and shuffled the mess hall around. Fixed all the issues, but I've left the armories and checkpoints as is. Genuinely think they're fine as-is.

Spookerton commented:
`No need for there to be a checkpoint in front of the armoury AND the little desk gimmick`

The checkpoint is for security to watch the armory, the little desk is for officers (not security) to hand out guns from and acts as a 2-step entry into the armory itself. They serve different purposes.

HeyBanditoz commented:
Could merge it for a week and revert it if people don't like it. Hold odd on merging torch map changes and merge some smaller map changes first. Or poll on the forums I guess.

BlueNexus commented:
Seeing as a lot of people seem to dislike this, a poll might be wise.

babydoll commented:
I think you may be biased in saying that.

babydoll commented:
There are still about a billion pipe leaks. Compile and fly around your map.

CakeQ commented:
fixing diffbot

CakeQ committed:
Moves security to deck 1, makes deck 3 more habitable

mapping additions

fixes
